src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -276,6 +279,12 @@ RoverPositionControl::control_position(const matrix::Vector2d &current_position,
 				mission_target_speed = _pos_sp_triplet.current.cruising_speed;
 			}
 
+			// Log target speed in global NED frame (since we don't always enforce the heading, it is in the direction we are headed)
+			const Vector3f target_vel_vec = mission_target_speed * ground_speed.unit();
+			_local_pos_setpoint.vx = target_vel_vec(0);
+			_local_pos_setpoint.vy = target_vel_vec(1);
```

Comment:
Since when using L1 guidance, we don't really have a 'velocity setpoint', I just used the target speed in the direction of unit vector of current vehicle's ground speed